gpkit/constraints/linked.py

```diff
@@ -4,6 +4,11 @@
 from .. import SignomialsEnabled
 
 
+class SubstitutionConflictError(Exception):
+    "Raised when linked variables have differing values."
+    pass
```

Comment:
I kind of like having the specific problem in the error name; do you think it's unclear?